whac-a-mole.py

```diff
@@ -3,6 +3,10 @@
 from time import *
 import threading
 import random
+from playsound import playsound
+

```

Comment:
Regarding PEP-8, please refer [this link](https://www.python.org/dev/peps/pep-0008/) and format the code accordingly. 
Install `pycodestyle` using your cmd (in windows), and test the script with it to confirm there are no PEP-8 style violations.
(This is important to learn, since most of the Python code written out there is formatted using this style only.)